package.json

```diff
@@ -2,16 +2,17 @@
   "author": "Roman Shtylman <shtylman@gmail.com>",
   "name": "localtunnel",
   "description": "expose localhost to the world",
-  "version": "1.5.1",
+  "version": "1.5.2",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git://github.com/shtylman/localtunnel.git"
   },
   "dependencies": {
-    "request": "2.11.4",
+    "debug": "0.7.4",
+    "openurl": "^1.1.0",
```

Comment:
I guess npm sort alphabetically the dependencies when I add the new ```openurl``` dependency. I also fix the spacing & indentation on ```bin/client``` source.